modules/auxiliary/gather/ldap_query.rb

```diff
@@ -309,7 +310,9 @@ def run
           entries = perform_ldap_query(ldap, filter, attributes)
           print_error("No entries could be found for #{datastore['QUERY_FILTER']}!") if entries.nil? || entries.empty?
         else
-          query = @loaded_queries[datastore['ACTION']]
+          require 'pry'; binding.pry
```

Comment:
And this 👀

Reply:
Thanks, fixed this and rebased the changes.